tools/testing/test_selections.py

```diff
@@ -127,7 +127,7 @@ def _query_changed_test_files() -> List[str]:
     return lines
 
 
-def get_reordered_tests(tests: List[ShardedTest]) -> List[ShardedTest]:
+def get_reordered_tests(tests: List[ShardedTest]) -> Tuple[List[ShardedTest], List[ShardedTest]]:
```

Comment:
@clee2000 just makes a change here in https://github.com/pytorch/pytorch/pull/100752, so I guess you need to do a quick rebase